integration_tests/test_str_01.py

```diff
@@ -37,6 +37,30 @@ def test_str_slice():
     # TODO:
     # assert a[0:5:-1] == ""
 
+def test_str_isalpha():
+    a:str = "helloworld"
+    b:str = "hj kl"
+    res: bool = a.isalpha()
+    res2: bool = b.isalpha()
+    assert res == True 
+    assert res2 == False
+   
+def test_str_title():
+    a:str = "hello world"
+    b:str = "hj'kl"
+    res: str = a.title()
+    res2: str = b.title()
+    assert res == "Hello World" 
+    assert res2 == "Hj'Kl"
+
+def test_str_istitle():
+    a:str = "Hello World"
+    b:str = "Hj'kl"
+    res: bool = a.istitle()
+    res2: bool = b.istitle()
+    assert res == True
+    assert res2 == False 
```

Comment:
Add test for empty string.
Also add " Hello", "HeLlo" or something similar

Reply:
Thanks, this was an edge case 
CPython `istitle and title` is not clear in documentation 
updated the code according to the response from cpython
```
>>> l="\tHello"
>>> l.istitle()
True
>>> l="\thello"
>>> l.istitle()
False
```